src/components/ShareCurrentScenarioButton/ShareCurrentScenarioButton.js

```diff
@@ -2,12 +2,13 @@
 // Licensed under the MIT license.
 
 import React from 'react';
+import PropTypes from 'prop-types';
 import { PermissionsGate, RolesEditionButton } from '@cosmotech/ui';
 
 import { ACL_PERMISSIONS } from '../../services/config/accessControl';
 import { useShareCurrentScenarioButton } from './ShareCurrentScenarioButtonHook';
 
-const ShareCurrentScenarioButton = () => {
+const ShareCurrentScenarioButton = ({ disabled }) => {
```

Comment:
IIRC we can avoid adding this prop because the component can access isDirty state directly bu `useFormState() `hook

Reply:
Done :slightly_smiling_face: